src/modules/remote-config/types.ts

```diff
@@ -3,4 +3,10 @@ import type { WalletNameFlag } from 'src/shared/types/WalletNameFlag';
 export interface RemoteConfig {
   extension_wallet_name_flags: Record<string, WalletNameFlag[]>;
   extension_uninstall_link: string;
+  loyalty_config: Partial<{
+    referrerXpPercent: number;
+    isEnabled?: boolean;
```

Comment:
It looks like it's already removed in Firebase, so we should probably remove it from this type too

Reply:
removed